lib/dialects/postgres/query-generator.js

```diff
@@ -375,7 +375,7 @@ var QueryGenerator = {
 
     var replacements = {
       table: this.quoteIdentifiers(tableName),
-      where: this.getWhereConditions(where),
+      where: this.getWhereConditions(where, null, model, options),
```

Comment:
@mickhansen `whereItemsQuery` will not work as `getWhereConditions` handle the perpending of primary key from model. Otherwise it generates queries like this

`WHERE 1 LIMIT 1 ` instead of `WHERE id = 1 LIMIT 1`